.github/workflows/build.yml

```diff
@@ -60,10 +78,10 @@ jobs:
         uses: actions/cache@v2.1.5
         id: npm-cache
         with:
-          path: ${{ steps.npm-cache-dir-path.outputs.dir }}
-          key: ${{ runner.os }}-npm-${{ hashFiles('**/package-lock.json') }}
+          path: ~/.npm
```

Comment:
What was wrong with the old approach? Much safer than hardcoded defaults imo.

Reply:
We only test on Linux environments, so the path is always the same. This is what's used in the cache action's documentation when running on Linux hosts.